SECURITY.md

```diff
@@ -0,0 +1,98 @@
+# Security Policy
+
+## Supported Versions
+
+The Jaeger project provides community support only for last minor version: bug fixes are released eiter as part of the next minor version or as an on-demand patch version. Independent of which version is next, all patch versions are cumulative, meaning that they represent the state of our `master` branch at the moment of the release. For instance, if the latest version is 1.19.0, bug fixes are released either as part of 1.20.0 or 1.19.1.
+
+Security fixes are given priority and might be enough to cause a new version to be released.
+
+## Reporting a Vulnerability
+
+_The following is a copy of the [Report a security issue](https://www.jaegertracing.io/report-security-issue/) page from our website. The website's version has precedence in case of conflicts._
+
+If you find something suspicious and want to report it, we'd really appreciate!
+
+### Ways to report
+
+* Prefer to always encrypt your message, no matter the channel you choose to report the issue
+* Contact us on our open chat room on [Gitter][gitter-room]
+* Send a message to [jaeger-tracing@googlegroups.com][mailing-list]
+* Merge request on GitHub: if you can, fork the affected repository and send us a pull request. We really prefer if you'd talk to us before, though, as our repositories are public and we would like to give a heads up to our users before disclosing it publicly
```

Comment:
I know these are the current instructions we use, but wondering if better not to include this item? If there is a security issue, then it should always be reported using a private/secure channel.

Reply:
I'll reword it a bit.